services/app-graphql/src/schema.graphql

```diff
@@ -1131,6 +1157,13 @@ type Rate {
     This value is used to generate the rateName
     """
     stateNumber: Int!
+     """
+    Information about who, when, and why this revision was unlocked.
+    Will be blank on the initial revision.
+    """
+    unlockInfo: UpdateInformation
+    "Information on who, when, and why this revision was submitted."
+    submitInfo: UpdateInformation
```

Comment:
I don't think we want `unlockInfo` and `submitInfo` at the `Rate` level. Was there something new you were trying to do here? Otherwise the `RateRevision` already has both.